From 0d4bb8d059721a4969f8c0267ce92c753ba1e521 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Fri, 15 Sep 2023 11:06:27 +0200
Subject: [PATCH 06/41] Omit linking to layout_proto_cc if protobuf linkage is
 shared

---
 tensorflow/python/BUILD | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tensorflow/python/BUILD b/tensorflow/python/BUILD
index 8a781badc68..5edfb0a4846 100644
--- a/tensorflow/python/BUILD
+++ b/tensorflow/python/BUILD
@@ -1382,7 +1382,6 @@ tf_python_pybind_extension(
     features = ["-layering_check"],
     deps = [
         ":pywrap_densor_device_headers",
-        "//tensorflow/dtensor/proto:layout_proto_cc",
         "//tensorflow/python/lib/core:pybind11_lib",
         "//tensorflow/python/lib/core:pybind11_status_headers",
         "//third_party/python_runtime:headers",  # buildcleaner: keep
