diff --git a/tensorflow/tools/pip_package/utils/tf_wheel.bzl b/tensorflow/tools/pip_package/utils/tf_wheel.bzl
index 057779e946e..5b4bc01b674 100644
--- a/tensorflow/tools/pip_package/utils/tf_wheel.bzl
+++ b/tensorflow/tools/pip_package/utils/tf_wheel.bzl
@@ -89,7 +89,7 @@ def _tf_wheel_impl(ctx):
     args.add("--project-name", WHEEL_NAME)
     args.add("--platform", _get_wheel_platform_name(
         ctx.attr.platform_name,
-        ctx.attr.platform_tag,
+        "arm64"
     ))
     args.add("--collab", str(WHEEL_COLLAB))
     args.add("--output-name", wheel_dir)
